qutebrowser/browser/downloads.py

```diff
@@ -131,7 +131,11 @@ def create_full_filename(basename, filename):
     encoding = sys.getfilesystemencoding()
     filename = utils.force_encoding(filename, encoding)
     basename = utils.force_encoding(basename, encoding)
-    if os.path.isabs(filename) and os.path.isdir(filename):
+
+    if filename.endswith('/'):
```

Comment:
Shouldn't this keep the `isabs` check? If the user enters `stuff/`, does that mean that we still want to prepend the download dir (and download to e.g. `/home/foo/Downloads/stuff/`)? Or should this raise an error?

Currently it seems like it uses the current working dir as a base then.